src/vs/editor/common/standaloneStrings.ts

```diff
@@ -25,6 +25,12 @@ export namespace AccessibilityHelpNLS {
 	export const tabFocusModeOffMsg = nls.localize("tabFocusModeOffMsg", "Pressing Tab in the current editor will insert the tab character. Toggle this behavior {0}.");
 	export const tabFocusModeOffMsgNoKb = nls.localize("tabFocusModeOffMsgNoKb", "Pressing Tab in the current editor will insert the tab character. The command {0} is currently not triggerable by a keybinding.");
 	export const showAccessibilityHelpAction = nls.localize("showAccessibilityHelpAction", "Show Accessibility Help");
+	export const saveAudioCueDisabled = nls.localize("saveAudioCueDisabled", "`audioCues.save` is disabled, so an alert will occur when a file is saved.");
```

Comment:
Just to check, what is an alert? Is it an in-product notification, or something like a Windows dialog (which makes its own sound)?

Reply:
It's an in product notification